server/controlers/index.js

```diff
@@ -0,0 +1,4 @@
+const router = require('express').Router();
+const login = require('./login_post.js');
+router.get('/login_post', login.post);
```

Comment:
I think you need router.post(), not router.get() for this route to apply to POST requests.

Reply:
Nevermind, sorry about that.  I was looking at an old file.